src/content/docs/infrastructure/amazon-integrations/manage-aws-data/aws-integrations-metrics.mdx

```diff
@@ -4574,6 +4595,162 @@ We no longer add these mappings, and we're committed to preserving all the new m
     </tr>
   </tbody>
 </table>
+    </Collapser>
+    <Collapser
+        id="namespace-entities"
+        title="AWS namespaces as New Relic entities"
+    >
+
+We generate New Relic [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic) for many AWS namespaces. When we generate New Relic entities for a namespace you can expect to:
```

Comment:
I think these two lines seem a bit repetitive. Maybe change the wording on the second one and make it more concise or something?